language/transaction-builder/generator/src/python3.rs

```diff
@@ -259,11 +406,33 @@ SCRIPT_ENCODER_MAP: typing.Dict[typing.Type[ScriptCall], typing.Callable[[Script
         writeln!(self.out, "}}\n")
     }
 
-    // TODO: Update to handle script function ABIs
-    fn output_decoder_map(&mut self, abis: &[TransactionScriptABI]) -> Result<()> {
+    fn output_script_function_encoder_map(&mut self, abis: &[ScriptFunctionABI]) -> Result<()> {
         writeln!(
             self.out,
-            "\nSCRIPT_DECODER_MAP: typing.Dict[bytes, typing.Callable[[Script], ScriptCall]] = {{"
+            r#"
+# pyre-ignore
```

Comment:
What changed that makes `pyre-ignore` necessary now?

Reply:
I was copying the `#pyre-ignore` from the declaration on the `TRANSACTION_SCRIPT_ENCODER_MAP` above that was pre-existing. I'll give it a shot to see if I can remove this.